mpf/devices/autofire.py

```diff
@@ -126,6 +126,8 @@ def disable(self, **kwargs):
 
     def _hit(self):
         """Rule was triggered."""
+        if not self._enabled:
```

Comment:
Good question what the expected behavior should be. In some cases (such as with kickbacks in an outlime) it would make sense to mark the playfield as active. Nevertheless it would also not hurt to not mark it active. I guess this is ok.

Reply:
Yeah, I wasn't sure if this would break any expectations. My assumption is that _most_ kickbacks would be on switches that explicitly defined `tags: playfield_active` so the autofire-playfield-active event is superfluous. But maybe I've just been wrong in tagging my outlanes as `playfield_active`?

And in any case, if the kickback is enabled and actually kicks the ball back, the *playfield_active* event will still post. If the kickback is disabled, presumably the outlane switch would send the ball to the drain so it's not important.